src/labels/CustomLabels.labels

```diff
@@ -26,13 +26,21 @@
     </labels>
     <labels>
         <fullName>DefaultHouseholdName</fullName>
-        <categories>Households, Household Naming</categories>
+        <categories>Account Naming</categories>
         <language>en_US</language>
         <protected>false</protected>
         <shortDescription>Default Household Name</shortDescription>
```

Comment:
That's actually the one for the household. I just categorized the three under Account Naming: https://github.com/SalesforceFoundation/HEDAP/blob/feature/216-acc-rec-types-change/src/labels/CustomLabels.labels#L19-L41